src/styles/calendar.scss

```diff
@@ -0,0 +1,62 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.sections {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $my-var; // Ширина календаря
```

Comment:
The variable `$my-var` is not a descriptive name. It should be renamed to reflect its purpose, such as `$calendar-width`. Remember to maintain meaningful variable names that convey their use in the code.